src/ur/workday.ur

```diff
@@ -0,0 +1,378 @@
+open Json
+(* open Bootstrap *)
+
+signature AUTH = sig
+    val token : transaction (option string)
+end
+
+(* Nonopaque type for all Workday IDs. *)
+type wid = string
+
+con worker = [
```

Comment:
Why have you bothered to define this constructor of kind `{Type}` rather than going right for the record type?  It seems you just need to keep writing many `$`s later, with no payoff.

Reply:
Vestige from an earlier version where I had mistakenly defined tables in the connector module and I used the `con` to define their schema and the type of API responses together. This is no longer the case, so I can change it back to a `type`. The other comments about this are similarly resolved.